client/my-sites/feature-upsell/store-upsell.jsx

```diff
@@ -123,7 +123,7 @@ class StoreUpsellComponent extends Component {
 							body={
 								<div className="google-voucher__initial-step">
 									<TipInfo
-										info={ 'Offer valid in US after spending the first $25 on Google Ads.' }
+										info={ 'Offer valid in US and CA after spending the first $25 on Google Ads.' }
```

Comment:
Same thing, we should extract the price.

Reply:
From top of file:
```
/*
 * This is just for english audience and is not translated on purpose, remember to add
 * translate() calls before removing a/b test check and enabling it for everyone
 */
```